packages/@dcl/dcl-rollup/index.ts

```diff
@@ -22,49 +24,54 @@ const colors = createColors({
   useColor: process.env.FORCE_COLOR !== '0' && !process.env.NO_COLOR
 })
 
-const WATCH =
-  process.argv.indexOf('--watch') !== -1 || process.argv.indexOf('-w') !== -1
+const args = arg({
+  '--watch': Boolean,
+  '-w': '--watch',
+  '--single': String,
+  '--production': Boolean,
+  '--project': String,
+  '-p': '--project'
+})
+
+const WATCH = !!args['--watch']
 
 // PRODUCTION === true : makes the compiler to prefer .min.js files while importing and produces a minified output
```

Comment:
Maybe add `const PREFER_MINIMIFIED_JS = PRODUCTION` and `const EXPORT_MINIFIED_JS = PRODUCTION`?